src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -1452,7 +1452,9 @@ public override ISymbol GetDeclaredSymbol(MemberDeclarationSyntax declarationSyn
                     // Global statements don't declare anything, even though they inherit from MemberDeclarationSyntax.
                     return null;
 
+#pragma warning disable CS0618 // Type or member is obsolete
                 case SyntaxKind.IncompleteMember:
+#pragma warning restore CS0618 // Type or member is obsolete
                     // Incomplete members don't declare any symbols.
                     return null;
```

Comment:
The only way to reach this is crafting a tree using the obsolete `SyntaxFactory.IncompleteMember` API. So I kept this as return null.

I think we will want to have a test whose behavior changes depending on the existence of this code path.